features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx

```diff
@@ -353,7 +353,7 @@ class ApiConsole extends React.Component {
             return 'API Not found !';
         }
 
-        const { authorizationHeader = 'Authorization' } = api;
+        const authorizationHeader = api.authorizationHeader ? api.authorizationHeader : 'Authorization';
```

Comment:
don't we need to read the server level config if the api level one is not there?

Reply:
[Here](https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java#L364), it is checking whether the server level configuration is enabled. Hence, if it is not set, it should have the default value